.github/workflows/ci.yml

```diff
@@ -200,21 +200,19 @@ jobs:
       - name: Build all lint dependencies
         run: make -j build-node-deps
 
-      # TODO: celestia has lint errors, fix in later PRs
-      # - name: Lint
-      #   if: matrix.test-mode == 'defaults'
-      #   uses: golangci/golangci-lint-action@v3
-      #   with:
-      #     version: v1.59.1
-      #     skip-cache: true
-
-      # - name: Custom Lint
-      #   if: matrix.test-mode == 'defaults'
-      #   run: |
-      #     go run ./linters ./...
+      - name: Lint
+        uses: golangci/golangci-lint-action@v3
+        with:
+          version: latest
+          skip-pkg-cache: true
+      - name: Custom Lint
+        run: |
+          go run ./linters ./...
 
       - name: Set environment variables
         run: |
+          mkdir -p target/tmp/deadbeefbee
```

Comment:
what is this lol?

Reply:
I wanted to include as little changes from upstream so I brought it back from celestia

Reply:
[d533bd0](https://github.com/EspressoSystems/nitro-espresso-integration/pull/561/commits/d533bd08cf470a1d4fc48ebac48d2d94fa21a65f)